documentation/asciidoc/topics/proc_creating_sa_tokens.adoc

```diff
@@ -30,6 +30,13 @@ the view role to the service account on **LON** as follows:
 $ oc policy add-role-to-user view system:serviceaccount:my-xsite:lon
 ----
 +
+. Add cluster-reader role to the service accounts (if using NodePor to expose the service)
```

Comment:
NodePort misprint

Reply:
I noticed this earlier but didn't comment as I plan on making a pass at xsite docs after things are ready on the eng side. @rigazilla could you please notify when things are ready and I'll add docs to this PR or in a separate PR if that's more efficient? thanks.